plugins/repository-s3/build.gradle

```diff
@@ -241,9 +258,9 @@ testClusters.yamlRestTest {
     setting 's3.client.integration_test_eks.region', { "us-east-2" }, IGNORE_VALUE
 
     // to redirect InstanceProfileCredentialsProvider to custom auth point
-    systemProperty "com.amazonaws.sdk.ec2MetadataServiceEndpointOverride", { "${-> fixtureAddress('s3-fixture', 's3-fixture-with-ec2', '80')}" }, IGNORE_VALUE
+    systemProperty "software.amazon.awssdk.sdk.ec2MetadataServiceEndpointOverride", { "${-> fixtureAddress('s3-fixture', 's3-fixture-with-ec2', '80')}" }, IGNORE_VALUE
```

Comment:
From https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/migration-client-changes.html, AWS SDK uses `aws.ec2MetadataServiceEndpoint`. This should be `aws.ec2MetadataServiceEndpoint`. This is a breaking change if anyone is trying to override this setting.

![Screenshot 2023-05-23 at 9 22 12 AM](https://github.com/opensearch-project/OpenSearch/assets/542335/67c6caf8-9cfa-4f7d-95c6-8d203adcc6d3)